### PR TITLE
updated oscillator strengths and gamma's for Si II 1206 and Si III 1260 re Issue #24

### PR DESCRIPTION
**Pull request by peeples**

### CODE REVIEW

**Comment by chummels** [comment]
I retriggered the tests, as they timed out for some reason.

Before this gets merged, we should double check that these new values match the NIST values (where we got the line data originally).  I don't understand how things got mixed up, and I want to assure there aren't more problems with other lines.

File: trident/data/line_lists/lines.txt [hunk 1]
```diff
@@ -129,7 +129,7 @@ Si II          1808.013000    2.540000e+06    2.490000e-03
 Si II          1309.276000    4.730000e+08    1.130000e-01     Si II* 1309
 Si II          1304.370000    3.040000e+09    1.090000e+00                
 Si II          1264.738000    8.300000e+08    1.940000e-01     Si II* 1265
-Si II          1260.422000    4.030000e+08    1.880000e-01                
+Si II          1260.422000    2.950000e+09    1.180000e+01
```
Comment (brittonsmith) [inline trident/data/line_lists/lines.txt]: Here's what I've found from [NIST](https://physics.nist.gov/cgi-bin/ASD/lines1.pl?spectra=Si+II&limits_type=0&low_w=1260&upp_w=1261&unit=0&de=0&java_window=3&java_mult=&format=0&line_out=0&en_unit=0&output=0&bibrefs=1&page_size=15&show_obs_wl=1&show_calc_wl=1&unc_out=1&order_out=0&max_low_enrg=&show_av=2&max_upp_enrg=&tsb_value=0&min_str=&A_out=0&f_out=on&S_out=on&intens_out=on&max_str=&allowed_out=1&forbid_out=1&min_accur=&min_intens=&conf_out=on&term_out=on&enrg_out=on&J_out=on&submit=Retrieve+Data): 2.57e+09 | 1.22e+00

File: trident/data/line_lists/lines.txt [hunk 2]
```diff
@@ -138,7 +138,7 @@ Si II          1023.700000    1.770000e+08    1.390000e-02     Si II* 1024
 Si II          1020.699000    8.910000e+07    1.390000e-02                
 Si II           992.683000    7.110000e+08    1.570000e-01      Si II* 993
 Si II           989.873000    6.810000e+08    2.000000e-01                
-Si III         1206.500000    3.000000e+07    2.150000e-03                
+Si III         1206.500000    2.480000e+09    1.630000e+00
```
Comment (brittonsmith) [inline trident/data/line_lists/lines.txt]: Results from [NIST](https://physics.nist.gov/cgi-bin/ASD/lines1.pl?spectra=Si+III&limits_type=0&low_w=1206&upp_w=1207&unit=0&submit=Retrieve+Data&de=0&java_window=3&java_mult=&format=0&line_out=0&en_unit=0&output=0&bibrefs=1&page_size=15&show_obs_wl=1&show_calc_wl=1&unc_out=1&order_out=0&max_low_enrg=&show_av=2&max_upp_enrg=&tsb_value=0&min_str=&A_out=0&f_out=on&S_out=on&intens_out=on&max_str=&allowed_out=1&forbid_out=1&min_accur=&min_intens=&conf_out=on&term_out=on&enrg_out=on&J_out=on): 2.55e+09 | 1.67e+00

File: trident/data/line_lists/lines.txt (hunk 1 above)
Comment (chummels) [reply trident/data/line_lists/lines.txt]: Weird--how did we get so far off?  I thought @devinsilvia pulled these straight from NIST, but maybe the NIST values have changed?

**Comment by chummels** [comment]
While @peeples has done a great job in correcting these two values in this PR, I honestly think that what we need to do is go through and totally vet the line list in its entirety.  If we got these two lines wrong, who knows how many others are wrong as well!

As @peeples points out in Issue #24 , these data come from Morton 2003.  I've checked with several other members of the CGM community, and it appears that everyone is using that dataset or updated datasets that have all been catalogued in the linetools python package by X Prochaska: https://github.com/linetools/linetools/blob/master/linetools/data/lines/morton03_table2.dat .  

So I think we need to just go through and totally check the lines.txt file to make sure all the values are correct.  This will necessitate an update of the answer tests, which is OK, but I want to avoid doing it over and over again as we update individual lines piecemeal.

File: trident/data/line_lists/lines.txt (hunk 1 above)
Comment (chummels) [reply trident/data/line_lists/lines.txt]: Upon further investigation, the two lines specified in this PR were misidentified by the script that queried NIST values and populated this list.  It appears at first look that there are no other differences between the line list and true NIST values.  I will continue to check this and report back.

Comment (qobilidop) [reply trident/data/line_lists/lines.txt]: I happen to be looking at this value. I'm comparing with the [mortan03 table](https://github.com/linetools/linetools/blob/master/linetools/data/lines/morton03_table2.dat), and find there's a typo: 1.18e+01 should be 1.18E+00.

Comment (brittonsmith) [reply trident/data/line_lists/lines.txt]: @qobilidop, can you issue a PR for that?

Comment (qobilidop) [reply trident/data/line_lists/lines.txt]: @brittonsmith, sure, will do.

**Pull request merged**